src/Service.Tests/Authorization/AuthorizationHelpers.cs

```diff
@@ -31,28 +33,35 @@ public static class AuthorizationHelpers
         /// <returns>AuthorizationResolver object</returns>
         public static AuthorizationResolver InitAuthorizationResolver(RuntimeConfig runtimeConfig)
         {
-            RuntimeConfigProvider runtimeConfigProvider = TestHelper.GetRuntimeConfigProvider(runtimeConfig);
+            MockFileSystem fileSystem = new();
+            fileSystem.AddFile(RuntimeConfigLoader.DEFAULT_CONFIG_FILE_NAME, new(runtimeConfig.ToJson()));
+            RuntimeConfigLoader loader = new(fileSystem);
+
+            RuntimeConfigProvider runtimeConfigProvider = TestHelper.GetRuntimeConfigProvider(loader);
             Mock<ISqlMetadataProvider> metadataProvider = new();
             Mock<ILogger<AuthorizationResolver>> logger = new();
             SourceDefinition sampleTable = CreateSampleTable();
             metadataProvider.Setup(x => x.GetSourceDefinition(TEST_ENTITY)).Returns(sampleTable);
-            metadataProvider.Setup(x => x.GetDatabaseType()).Returns(DatabaseType.mssql);
+            metadataProvider.Setup(x => x.GetDatabaseType()).Returns(DatabaseType.MSSQL);
 
             string? outParam;
             Dictionary<string, Dictionary<string, string>> _exposedNameToBackingColumnMapping = CreateColumnMappingTable();
             metadataProvider.Setup(x => x.TryGetBackingColumn(It.IsAny<string>(), It.IsAny<string>(), out outParam))
                               .Callback(new metaDataCallback((string entity, string exposedField, out string? backingColumn) => _ = _exposedNameToBackingColumnMapping[entity].TryGetValue(exposedField, out backingColumn)))
                               .Returns((string entity, string exposedField, string? backingColumn) => _exposedNameToBackingColumnMapping[entity].TryGetValue(exposedField, out backingColumn));
 
-            return new AuthorizationResolver(runtimeConfigProvider, metadataProvider.Object, logger.Object);
+            metadataProvider.Setup(x => x.GetEntityName(It.IsAny<string>()))
+                .Returns((string entity) => entity);
```

Comment:
were tests failing with out this?

Reply:
Yes. I did a refactor that means we call that now to get the entity name from the GraphQL type, so the tests need it.